src/components/molecules/ZopaFooter/ZopaFooter.tsx

```diff
@@ -31,13 +31,15 @@ const StyledLink = styled(Link)`
 export interface FooterProps extends HTMLAttributes<HTMLDivElement> {
   baseUrl?: string;
   renderLink?: (props: LinkProps) => React.ReactNode;
+  mainCustomLegalCopy?: string;
```

Comment:
Maybe it will be nice to add a js doc explaining this is flexible depending on the partner